README.md

```diff
@@ -688,6 +688,22 @@ client.without_caching do
 end
 ```
 
+If you prefer to opt in to caching on a per-request, you can do so by using .with_caching and
+setting the use_cache config option to false:
```

Comment:
`use_cache`

Reply:
Once again, not 100% sure what you're recommending - you just want me to just add backticks around use_cache?

Reply:
Exactly!